.devcontainer/Dockerfile

```diff
@@ -16,6 +16,7 @@ FROM ghcr.io/eclipse-ankaios/devcontainer-base:0.9.1
 
 ARG USERNAME=vscode
 
+RUN apt-get update && apt-get -y install musl-tools
```

Comment:
I think finally we need to move this to Dockerfile.base as otherwise the CI build will continue to fail.